src/App.js

```diff
@@ -1,14 +1,24 @@
 import TodoItem from "./components/TodoItem/TodoItem";
 import InputTodo from "./components/InputTodo";
 import FilterTodos from "./components/FilterTodos";
+import Modal from "./components/Modal";
+import ModalTwo from "./components/ModalTwo";
 
-import { useState, useEffect } from "react";
+import { useState, useEffect, useRef } from "react";
 
-function App() {
+export default function App() {
   const [value, setValue] = useState("");
   const [todos, setTodos] = useState([]);
   const [status, setStatus] = useState("all");
   const [filteredTodos, setFilteredTodos] = useState([]);
+  const [openModal, setOpenModal] = useState({
+    message: "",
+    isLoading: false,
+  });
```

Comment:
const [isModalShown, setIsModalShown] = useState(false)

Use that function signature instead, you don't need to set a message (string) on the state, pass the message as a prop instead